fastlane/lib/fastlane/actions/update_fastlane.rb

```diff
@@ -56,32 +57,49 @@ def self.run(options)
 
         update_needed.each do |tool_info|
           tool = tool_info[0]
+          gem_version = tool_info[1]
           local_version = Gem::Version.new(highest_versions[tool].version)
           update_url = FastlaneCore::UpdateChecker.generate_fetch_url(tool)
-          latest_version = FastlaneCore::UpdateChecker.fetch_latest(update_url)
-          UI.message("Updating #{tool} from #{local_version} to #{latest_version} ... 🚀")
+          latest_official_version = FastlaneCore::UpdateChecker.fetch_latest(update_url)
+
+          if options[:nightly]
+            UI.message("Updating #{tool} from #{local_version.to_s.yellow} to nightly build #{gem_version.to_s.yellow}... (last official release #{latest_official_version.to_s.yellow}) 🚀")
+          else
+            UI.message("Updating #{tool} from #{local_version.to_s.yellow} to #{latest_official_version.to_s.yellow}... 🚀")
+          end
 
           # Approximate_recommendation will create a string like "~> 0.10" from a version 0.10.0, e.g. one that is valid for versions >= 0.10 and <1.0
-          updater.update_gem tool, Gem::Requirement.new(local_version.approximate_recommendation)
+          updater.update_gem(tool, Gem::Requirement.new(gem_version))
```

Comment:
Do you know if there is a way to download pre-releases in that action? I wasn't sure how else to solve it.